tests/js/spec/redux/actions/workBatch.spec.jsx

```diff
@@ -0,0 +1,106 @@
+import configureStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import moxios from 'moxios';
+
+import {
+  WORK_BATCHES_GET_REQUEST,
+  WORK_BATCHES_GET_SUCCESS,
+  WORK_BATCHES_GET_FAILURE,
+  WORK_BATCHES_TOGGLE_SELECT_ALL,
+  WORK_BATCH_TOGGLE_SELECT,
+  workBatchesGetRequest,
+  workBatchesGetSuccess,
+  workBatchesGetFailure,
+  workBatchesGet,
+  workBatchToggleSelect,
+  workBatchesToggleSelectAll,
+} from 'app/redux/actions/workBatch';
+
+const middlewares = [thunk];
+const mockStore = configureStore(middlewares);
+
+describe('workBatch redux actions', function() {
+  beforeEach(function() {
+    moxios.install();
+  });
+
+  afterEach(function() {
+    moxios.uninstall();
+  });
+
+  const mockWorkBatch = {
+    id: 4,
+    name: 'Test3',
+    organization: 1,
+    handler: 'somehandler2',
+    created: '2019-06-12T13:07:13.490Z',
+    extra_fields: '',
+    num_comments: 0,
```

Comment:
Ahh, we have python names for these fields. I'll have to fix that later :)

Reply:
Ok :)